dev/bootstrap.cfg

```diff
@@ -10,6 +10,7 @@ puppetlabs.services.legacy-routes.legacy-routes-service/legacy-routes-service
 puppetlabs.services.puppet-admin.puppet-admin-service/puppet-admin-service
 puppetlabs.trapperkeeper.services.authorization.authorization-service/authorization-service
 puppetlabs.services.versioned-code-service.versioned-code-service/versioned-code-service
+puppetlabs.trapperkeeper.services.scheduler.scheduler-service/scheduler-service
```

Comment:
Hmm, I thought this was the bootstrap.cfg that was used for tests but I have a few failures that say they can't find `:SchedulerService`. Not sure if that's because it isn't being loaded, or if I'm missing adding it into some sort of test utils setup for trapperkeeper for the tests?